docs/send-data/opentelemetry-collector/remote-management/source-templates/postgresql/index.md

```diff
@@ -0,0 +1,117 @@
+---
+slug: /send-data/opentelemetry-collector/remote-management/source-templates/postgresql
+title: PostgreSQL Source Template
+sidebar_label: PostgreSQL
+description: Learn about the Sumo Logic PostgreSQL source template for OpenTelemetry.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head> 
+  <meta name="robots" content="noindex" />
+</head>
```

Comment:
Since this doc is present in the sidebars, I'd remove this code snippet, which is only intended for beta and/or hidden docs. It blocks it from Algolia and Google search.

Reply:
@sumoanema

Reply:
Done. Thanks for pointing that out @kimsauce